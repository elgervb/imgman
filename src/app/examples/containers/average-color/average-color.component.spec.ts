import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { AverageColorComponent } from './average-color.component';

describe('AverageColorComponent', () => {
  let component: AverageColorComponent;
  let fixture: ComponentFixture<AverageColorComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [AverageColorComponent]
    })
      .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(AverageColorComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();

    expect(fixture).toMatchSnapshot();
  });
});
