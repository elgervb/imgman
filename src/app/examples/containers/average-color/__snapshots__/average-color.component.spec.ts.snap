// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AverageColorComponent should create 1`] = `
<app-average-color
  image={[Function ElementRef]}
>
  <img
    alt="ugly dog"
    src="/assets/ugly-dog.png"
  /><p
    class="color"
  >
    Average color: 
  </p>
</app-average-color>
`;
